tip.js

```diff
@@ -0,0 +1,76 @@
+// var unified = require('unified');
```

Comment:
Should also be moved to a subdirectory, see https://github.com/webpack/webpack.js.org/pull/2046#discussion_r181557862
Need clean up, and tests would be cool!